src/bundles/peer-locations.js

```diff
@@ -114,6 +124,44 @@ const parseLatency = (latency) => {
   return value
 }
 
+function getPublicIP (identity) {
+  if (!identity) return
```

Comment:
The callers of this don't explicitly handle a null return value. Is there a reasonable expectation there'll be no identity at this point?

Reply:
It should, usually. Just updated the function!